src/tests/helpers/purchase-operation-helper.test.ts

```diff
@@ -165,7 +165,7 @@ describe("PurchaseOperationHelper", () => {
     const getCheckoutStatusResponse: CheckoutStatusResponse = {
       operation: {
         status: CheckoutSessionStatus.Succeeded,
-        isExpired: false,
+        is_expired: false,
```

Comment:
Was this field ever exposed outside of the sdk?
I am guessing no.

Reply:
it's not used but we fixed with Toni anyway, since it was not parsed from the json response